test/hotspot/jtreg/runtime/Thread/AsyncExceptionTest.java

```diff
@@ -48,14 +48,16 @@ public class AsyncExceptionTest extends Thread {
     private boolean receivedThreadDeathinInternal1 = false;
     private boolean receivedThreadDeathinInternal2 = false;
 
+    public static native int stopThread(Thread thread);
+
     @Override
     public void run() {
         try {
             internalRun1();
         } catch (ThreadDeath td) {
             throw new RuntimeException("Catched ThreadDeath in run() instead of internalRun2() or internalRun1(). receivedThreadDeathinInternal1=" + receivedThreadDeathinInternal1 + "; receivedThreadDeathinInternal2=" + receivedThreadDeathinInternal2);
```

Comment:
Drive-by comment: the exception messages mean the lines are 240+ characters line and make it impossible to see changes when using side-by-side diffs. Maybe someday it should be trimming down to something sane.